docs/core/logging.md

```diff
@@ -650,6 +653,9 @@ Mapped Diagnostic Context (MDC) is essentially a Key-Value store. It is supporte
 ???+ warning "Custom keys stored in the MDC are persisted across warm invocations"
     Always set additional keys as part of your handler method to ensure they have the latest value, or explicitly clear them with [`clearState=true`](#clearing-state).
 
+???+ warning "Do not add reserved keys to MDC"
```

Comment:
I googled MDC, but could you share more as for why we are not applying this same key filtering logic to keys set to MDC?

Also the product name is `Powertools for AWS`; if you don't want to use this just refer to it as `Logger` in this type of context.

Reply:
I sent an update addressing this comment.